### PR TITLE
2.0 readme examples

### PR DESCRIPTION
**Pull request by Smitsel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,19 +15,18 @@ To use the Mollie API client, the following things are required:
 + Now you're ready to use the Mollie API client in test mode.
 + Follow [a few steps](https://www.mollie.com/dashboard/?modal=onboarding) to enable payment methods in live mode, and let us handle the rest.
 + PHP >= 5.6
-+ PHP cURL extension
 + Up-to-date OpenSSL (or other SSL/TLS toolkit)
```
Comment (Smitsel) [inline README.md]: Would one still need OpenSSL to work with this API @willemstuursma ?

Comment (willemstuursma) [reply README.md]: Yes, this is needed for Guzzle.

File: examples/02-webhook-verification.php [hunk 2]
```diff
@@ -28,9 +28,25 @@
         /*
          * At this point you'd probably want to start the process of delivering the product to the customer.
          */
-    } elseif ($payment->isOpen() == false) {
+    } elseif ($payment->isOpen() == true) {
         /*
-         * The payment isn't paid and isn't open anymore. We can assume it was aborted.
+         * The payment is open.
+         */
+    } elseif ($payment->isPending() == false) {
```
Comment (Daanvm) [inline examples/02-webhook-verification.php]: Your checks are inverted. I would also remove the `== true`.

File: examples/04-ideal-payment.php [hunk 3]
```diff
@@ -18,11 +18,11 @@
...
      */
     if ($_SERVER["REQUEST_METHOD"] != "POST") {
-        $issuers = $mollie->issuers->all();
+        $method = $mollie->methods->get(PaymentMethod::IDEAL, ["include" => "issuers"]);
 
         echo '<form method="post">Select your bank: <select name="issuer">';
 
-        foreach ($issuers as $issuer) {
+        foreach ($method->issuers as $issuer) {
             if ($issuer->method == PaymentMethod::IDEAL) {
```
Comment (Daanvm) [inline examples/04-ideal-payment.php]: This check is no longer needed now the issuers are coming as part of the payment method request.

File: examples/06-list-activated-methods.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+     */
+    require "initialize.php";
+    /*
+     * Get all the activated methods for this API key.
+     */
+    $methods = $mollie->methods->all();
+    foreach ($methods as $method)
+    {
+        echo '<div style="line-height:40px; vertical-align:top">';
+        echo '<img src="' . htmlspecialchars($method->image->size1x) . '"> ';
```
Comment (Daanvm) [inline examples/06-list-activated-methods.php]: Would it be nice to also include the `@2x` version of the image in a `srcset`?

File: README.md [hunk 5]
```diff
@@ -119,16 +119,17 @@ definitive. Refunds are only supported for iDEAL, credit card, Bancontact/Mister
 be refunded through our API at the moment.
 
 ```php
-$payment = $mollie->payments->get($payment->id);
+$payment = $mollie->payments->get($payment->id);
```
Comment (Daanvm) [inline README.md]: Unneeded spaces.

File: examples/06-list-activated-methods.php [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+     */
+    require "initialize.php";
+    /*
+     * Get all the activated methods for this API key.
+     */
+    $methods = $mollie->methods->all();
+    foreach ($methods as $method)
+    {
+        echo '<div style="line-height:40px; vertical-align:top">';
+        echo '<img src="' . htmlspecialchars($method->image->size1x) . '" srcset="' . htmlspecialchars($method->image->size2x) .' 2x"> ';
```
Comment (Daanvm) [inline examples/06-list-activated-methods.php]: I don't know a lot about the `srcset` attribute, but I did a quick test and it seems that you have to add the `1x` to the `srcset` too. You also have to set the width of the image to 40 explicitly.

Comment (Daanvm) [reply examples/06-list-activated-methods.php]: Nevermind, now I test it again it works like you did it.

**Comment by Daanvm** [comment]
LGTM

**Pull request merged**